src/pages/articles/ArticleCard.jsx

```diff
@@ -1,5 +1,29 @@
+import { useState } from "react"
+import { updateVotes } from "../../components/utils/api"
+
 const ArticleCard = (props) => {
-    const {title, author, topic, created_at, votes, article_img_url, comment_count} = props.article
+    const [article, setArticle] = useState(props.article)
```

Comment:
once this is refactored to have an api request per article, we will also need a loading state here